<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetTrianglesDrawable</class>
 <widget class="QWidget" name="WidgetTrianglesDrawable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>373</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>50</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Surface mesh renderer</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelDrawableName">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Drawable</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="comboBoxDrawables">
       <property name="toolTip">
        <string>Select the drawable to be visualized</string>
       </property>
       <item>
        <property name="text">
         <string>not available</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelVisible">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Visible</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QCheckBox" name="checkBoxVisible">
         <property name="toolTip">
          <string>Show/Hide this drawable</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>127</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelPhongShading">
       <property name="text">
        <string>Phong</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QCheckBox" name="checkBoxPhongShading">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Enable/Disable Phong shading</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>127</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelLighting">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Lighting</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QComboBox" name="comboBoxLightingOptions">
       <property name="toolTip">
        <string>Select how lighting is implemented</string>
       </property>
       <item>
        <property name="text">
         <string>front and back</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>front only</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>disabled</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelColorScheme">
       <property name="text">
        <string>Coloring</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QComboBox" name="comboBoxColorScheme">
       <property name="toolTip">
        <string>Select how color is assigned to each face</string>
       </property>
       <item>
        <property name="text">
         <string>uniform color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>use color property</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>19</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="labelColor">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="spacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelDefaultColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>default</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonDefaultColor">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Click to set a uniform color for this drawable</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../Resources/icons/transparency.png</normaloff>../Resources/icons/transparency.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelBackColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>back </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxBackColor">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Enable/Disable distinct coloring for the back side of this drawable</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonBackColor">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Click to set the backside color for this drawable</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../Resources/icons/transparency.png</normaloff>../Resources/icons/transparency.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>15</width>
           <height>22</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="labelTexture">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Texture</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEditTextureFile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>The image file of the texture</string>
         </property>
         <property name="text">
          <string>not available</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonTextureFile">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Click to choose a texture image</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>48</width>
         <height>22</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="labelTextureRepeat">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>repeat</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="spinBoxTextureRepeat">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>How many times this texture is repeated?</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>99</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxTextureFractionalRepeat">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Fractional repeat (100 fractional repeat is equivalent to 1 repeat)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelHighlight">
       <property name="text">
        <string>Highlight</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QCheckBox" name="checkBoxHighlight">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="8" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QSpinBox" name="spinBoxHighlightLow">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>The lower bound of the triange range to be highlighted.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxHighlightHigh">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>The upper bound of the triange range to be highlighted.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelOpacity">
       <property name="text">
        <string>Opacity</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="QSlider" name="horizontalSliderOpacity">
       <property name="toolTip">
        <string>The opacity of the surface</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxDrawables</tabstop>
  <tabstop>checkBoxVisible</tabstop>
  <tabstop>checkBoxPhongShading</tabstop>
  <tabstop>comboBoxLightingOptions</tabstop>
  <tabstop>comboBoxColorScheme</tabstop>
  <tabstop>toolButtonDefaultColor</tabstop>
  <tabstop>checkBoxBackColor</tabstop>
  <tabstop>toolButtonBackColor</tabstop>
  <tabstop>lineEditTextureFile</tabstop>
  <tabstop>toolButtonTextureFile</tabstop>
  <tabstop>spinBoxTextureRepeat</tabstop>
  <tabstop>spinBoxTextureFractionalRepeat</tabstop>
  <tabstop>checkBoxHighlight</tabstop>
  <tabstop>spinBoxHighlightLow</tabstop>
  <tabstop>spinBoxHighlightHigh</tabstop>
  <tabstop>horizontalSliderOpacity</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
